visualization/app/codeCharta/util/usageDataTracker.spec.ts

```diff
@@ -39,40 +35,27 @@ describe("UsageDataTracker", () => {
 	})
 
 	function mockTrackingToBeAllowed() {
-		jest.spyOn(EnvironmentDetector, "isStandalone").mockReturnValue(true)
 		jest.spyOn(FilesHelper, "isSingleState").mockReturnValue(true)
 		jest.spyOn(FilesHelper, "getVisibleFileStates").mockReturnValue([singleFileState])
 	}
 
 	describe("trackMetaUsageData", () => {
-		const setItemMock = jest.fn()
-		CodeChartaStorage.prototype.setItem = setItemMock
-
-		afterAll(() => {
-			jest.resetAllMocks()
-			jest.unmock("./codeChartaStorage")
+		beforeEach(() => {
+			jest.spyOn(window.localStorage.__proto__, "setItem")
+			window.localStorage.__proto__.setItem = jest.fn()
 		})
-
-		it("should not track in web version, in multi/delta mode or for more than one uploaded file", () => {
-			jest.spyOn(EnvironmentDetector, "isStandalone").mockReturnValue(false)
-			jest.spyOn(FilesHelper, "isSingleState").mockReturnValue(true)
-			jest.spyOn(FilesHelper, "getVisibleFileStates").mockReturnValue([{} as FileState])
-
-			trackMapMetaData(stateStub.files)
-
-			jest.spyOn(EnvironmentDetector, "isStandalone").mockReturnValue(true)
+		it("should not track in multi/delta mode or for more than one uploaded file", () => {
```

Comment:
How can we handle it when we don't have a single mode anymore?

Reply:
Let's adress this issue in #2656. I did not want to fix it in this issue to not going out of scope.